Lombiq.NodeJs.Extensions.Samples/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "private": true,
+  "devDependencies": {
+    "nodejs-extensions": "link:..\\Lombiq.NodeJs.Extensions",
+    "npm-run-all": "^4.1.5"
+  },
+  "scripts": {
+    "dotnet-prebuild": "pnpm build",
+    "dotnet-postclean": "pnpm clean",
+    "build": "npm-run-all --parallel build:styles build:styles:args",
+    "build:styles": "npm explore nodejs-extensions -- pnpm build:styles",
+    "build:styles:args": "npm explore nodejs-extensions -- pnpm build:styles:args --source=NonDefaultScssFolder --target=wwwwNonDefaultCssFolder",
+    "clean": "npm-run-all --parallel clean:styles clean:styles:args",
+    "clean:styles": "npm explore nodejs-extensions -- pnpm clean:styles",
+    "clean:styles:args": "npm explore nodejs-extensions -- pnpm clean:styles:args --target=wwwwNonDefaultCssFolder",
+    "watch": "npm-run-all --parallel watch:styles watch:styles:args",
```

Comment:
I get this error:

![image](https://user-images.githubusercontent.com/1976647/155865322-f0683a3f-a32e-4a14-b5c8-872a044bc71e.png)

`watch:styles` works.

Reply:
Hm, I don't. Please try again with the latest updates.